core/utils.py

```diff
@@ -333,7 +333,7 @@ def get_random_int(upper_bound: int) -> int:
     Returns:
         int. Randomly generated integer less than the upper_bound.
     """
-    assert upper_bound >= 0 and isinstance(upper_bound, int)
+    assert upper_bound >= 0 and isinstance(upper_bound, int), 'Only positive integers allowed' # pylint: disable=line-too-long
```

Comment:
Can you break this line with parentheses? Alternatively, use `raise AssertionError(...)` instead of `assert` to avoid the comma syntax

ditto elsewhere

Reply:
Done